cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/BaseWidget.kt

```diff
@@ -5,8 +5,7 @@ import com.cognifide.cogboard.config.service.WidgetRuntimeService
 import io.vertx.core.Vertx
 import io.vertx.core.eventbus.MessageConsumer
 import io.vertx.core.json.JsonObject
-import java.util.Timer
-import java.util.TimerTask
+import java.util.*
```

Comment:
Linter will definitely not like this wildcard.

Reply:
See here:
https://travis-ci.org/Cognifide/cogboard/builds/622682185#L301

Reply:
THX Mikołaj this change got there by mistake. I need to check my IDE import setup.

Reply:
I had a default setting in IDE that did this. You may check it out under:
`Settings` -> `Editor` -> `Code Style` -> `Kotlin` -> `Imports (tab)` -> `Packages to use imports with '*' property`